app.ts

```diff
@@ -15,6 +15,9 @@ addEventListener('fetch', (event: any) => {
 
 const handleRequest = async function (event: any) {
   const {request} = event;
+  if (!request.url.includes("reserveNFT") && !request.url.includes("staging")) {
```

Comment:
`!request.url.includes("staging")` we don't want to only protect staging but not production. Let's discuss offline for this.